etcdmain/help.go

```diff
@@ -216,10 +216,5 @@ Unsafe feature:
 CAUTIOUS with unsafe flag! It may break the guarantees given by the consensus protocol!
 
 TO BE DEPRECATED:
```

Comment:
Do we want to keep it empty or don't display it at all? Optionally, we can leave a comment in the code that add it back as needed?

Reply:
SGTM